apsuite/commisslib/measure_orbit_stability.py

```diff
@@ -125,6 +126,15 @@ def switching_freq(self):
     def switching_freq(self, val):
         self._switching_freq = val
 
+    @property
+    def frequency_band_filter(self):
+        """."""
+        return self._frequency_band_filter
+
+    @frequency_band_filter.setter
+    def frequency_band_filter(self, val):
```

Comment:
this name is not intuitive. This is not a filter, it is just bandwidth.

Reply:
With the new switching filtering method this attribute is no longer necessary and I deleted it.